en/ros2/user_guide.md

```diff
@@ -751,6 +751,115 @@ will generate topics under the namespaces:
 
 :::
 
+## PX4 ROS 2 Service Servers
+
+<Badge type="tip" text="PX4 v1.15" />
+
+PX4 uXRCE-DDS middleware supports [ROS 2 services](https://docs.ros.org/en/jazzy/Concepts/Basic/About-Services.html).
+Services are remote procedure calls, from one node to another, that return a result.
+
+A service server is the entity that will accept a remote procedure request, perform some computation on it, and return the result.
+They simplify communication between ROS 2 nodes and PX4 by encapsulating and associating the request and response topics, and ensuring that replies are only returned to the specific requesting user.
+This is much easier that publishing the request, subscribing to the reply, and filtering out any unwanted responses.
+
+The service servers that are built into the PX4 [uxrce_dds_client](../modules/modules_system.md#uxrce-dds-client) module include:
+
+- `/fmu/vehicle_command` (definition: [`px4_msgs::srv::VehicleCommand`](https://github.com/PX4/px4_msgs/blob/main/srv/VehicleCommand.srv).)
```

Comment:
1. What is the terminology/name for `/fmu/vehicle_command`? I.e. `/fmu/out/` is a publication, and `fmu/in` is a subscription that you an publish to. Is it just a "server"? Is it a "topic" or a "message"?

2. What is the name for `px4_msgs::srv::VehicleCommand` ? YOu called it a service Type - where does "Type" come from? I have called it a definition here, because that's what is linked.

[Ultimately I'm asking if this line is OK!)

3. `VehicleCommand.srv` is a service server message definition. Should we be auto-generating docs for this, as we do for normal uorb messages?

Reply:
Thank you very much for this. I'm going to email myself with a link and think about it a bit more.